content/docs/components/central-dashboard.md

```diff
@@ -0,0 +1,11 @@
++++
+title = "Central Dashboard"
```

Comment:
Also, we have an existing page about the Kubeflow UI here:
https://www.kubeflow.org/docs/other-guides/accessing-uis/

I'd recommend merging the content of that existing page into this page. As you'll need to delete the existing page, you should also add a redirect pointing to the new page.

I'm happy to do the merge, if you'd prefer not to do it. In that case, please raise an issue in `kubeflow/website` and assign it to me.

Reply:
Done, issue made @ #1549